tests/lib/eslint.js

```diff
@@ -2451,6 +2454,22 @@ describe("eslint", function() {
                 assert.equal(messages[0].message, "Cannot find module 'esprima-fbxyz'");
             });
 
+            it("should strip leading line: prefix from parser error", function() {
+                var parser = parserFixtures + "line-error.js";
```

Comment:
just to be safe why not use `path.join` here since you already have path in context.